diff -Naur cfengine-3.10.5.orig/configure.ac cfengine-3.10.5/configure.ac
--- cfengine-3.10.5.orig/configure.ac	2018-11-16 01:50:14.000000000 -0600
+++ cfengine-3.10.5/configure.ac	2019-05-05 13:08:40.252013842 -0500
@@ -515,8 +515,11 @@
       [],
       [if test "x$with_libacl" != xcheck; then AC_MSG_ERROR(Cannot find libacl library); fi])
     AC_CHECK_HEADERS([acl.h sys/acl.h acl/libacl.h],
-      [libacl_header_found=yes],
-      [if test "x$with_libacl" != xcheck; then AC_MSG_ERROR(Cannot find libacl header files); fi])
+      [libacl_header_found=yes])
+    if test "x$libacl_header_found" != "xyes" && test "x$with_libacl" != xcheck;
+    then
+      AC_MSG_ERROR(Cannot find libacl library headers);
+    fi
   ])
 fi
 
