From fab9e5815766d5ed0434b0ab82f17d2a11c9ad6d Mon Sep 17 00:00:00 2001
From: Chris Rorvick <chris@rorvick.com>
Date: Wed, 31 Aug 2016 23:55:21 -0500
Subject: [PATCH] fix printf argument types

---
 outguess.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/outguess.c b/outguess.c
index 557be9e..c0448b6 100644
--- a/outguess.c
+++ b/outguess.c
@@ -693,7 +693,7 @@ do_embed(bitmap *bitmap, u_char *filename, u_char *key, u_int klen,
 	if (bitmap->maxcorrect && correctlen > bitmap->maxcorrect) {
 		fprintf(stderr, "steg_embed: "
 			"message larger than correctable size %d > %d\n",
-			correctlen, bitmap->maxcorrect);
+			(int)correctlen, (int)bitmap->maxcorrect);
 		exit(1);
 	}
 
@@ -975,7 +975,7 @@ main(int argc, char **argv)
 			if (bitmap.maxcorrect)
 				fprintf(stderr,
 					"Correctable message size: %d bits, %0.2f%%\n",
-					bitmap.maxcorrect,
+					(int)bitmap.maxcorrect,
 					(float)100*bitmap.maxcorrect/bitmap.bits);
 		}
 
-- 
2.9.3

